SLAM/EKFSLAM/ekf_slam.py

```diff
@@ -29,8 +29,8 @@
 def ekf_slam(xEst, PEst, u, z):
     # Predict
     S = STATE_SIZE
-    xEst[0:S] = motion_model(xEst[0:S], u)
     G, Fx = jacob_motion(xEst[0:S], u)
+    xEst[0:S] = motion_model(xEst[0:S], u)
```

Comment:
Switched the order of the two statements, because the `jacob_motion` should use the expected robot pose **prior** to executing the motion command. 

Corresponding reference from the Probabilistic robotics book: 

![Bildschirmfoto 2020-03-24 um 13 06 51](https://user-images.githubusercontent.com/49170155/77423871-6e565c00-6dd0-11ea-8e08-22f25d914cb5.png)

Reply:
Oh good catch!!